### DISCUSSION TITLE
Upgrading WiX3 to WiX5

### DISCUSSION THREAD

**Discussion by vivek1303** [wixtoolset/issues#8613] [Questions]
Hi team, I was working around with creation of MSI installer for electron application in windows for which I was following this blog [MSI installation for electron](https://ourcodeworld.com/articles/read/927/how-to-create-a-msi-installer-in-windows-for-an-electron-framework-application). Here, it was pointing to WiX3 for the installation and I got the .exe installer for WiX3 in wix3 repo, however I was looking for the same installer .exe for WiX 5.0.0, it is not available. Is there a way we can get all the toolset which were generated through .exe installer for WiX3 for WiX5 as well?

**Comment by bevanweiss** [comment]
You'll want to read up on the Intro (which is for the newer version(s))
https://wixtoolset.org/docs/intro/

And then you'll want to read through the v3->v4 and the v4->v5 info in this same location.
